### ISSUE TITLE
Text from external file .txt

### ISSUE THREAD

**Issue by marteloudini** [Swordfish90/cool-retro-term#863]
can you add simple mode, to create typewriter screen and display all text from external .txt

Thanks.

**Comment by normanr** [comment]
You probably want to use one of the various tools like script or ttyrec that can record and replay terminal information.

**Comment by marteloudini** [comment]
> You probably want to use one of the various tools like script or ttyrec that can record and replay terminal information.

no, i want use cool-retro-term and simples externals files .txt into directory example : chap01/text001 ; chap01/text002 ; chap02/text001 ; chap02/text002 ; chap02/text003 etc..
and all files view into cool-retro-term easely.

**Comment by normanr** [comment]
Like using `cat chap01/text001`? Or maybe one of the many suggestions at something like https://superuser.com/questions/526242/cat-file-to-terminal-at-particular-speed-of-lines-per-second

**Comment by marteloudini** [comment]
> Like using `cat chap01/text001` (or maybe `less`)? Or maybe one of the many suggestions at something like https://superuser.com/questions/526242/cat-file-to-terminal-at-particular-speed-of-lines-per-second

interresting, thanks i look that.

i run cool-retro-term, after i call un perl script catLps.pl 
`#!/usr/bin/perl -w

use strict;
use Time::HiRes qw|time|;

my $start=time;
my $lps=300;

$lps=shift @ARGV if @ARGV && $ARGV[0]=~/^(\d+)$/;
my $sleepPerLine=1/$lps;

print &&
    select undef,undef,undef,($start + $sleepPerLine*$. - Time::HiRes::time)
    while <>`

Usage:
catLps.pl [lps] [file] [file]...

First argument lps is optional line per seconds numeric argument (default: 300)
Note: if filename is only numeric, you may have to specifiy them with path: ./3.
Like cat this could pass files given as argument and/or standard input

So we could:
TIMEFORMAT='%R' 
time seq 1 100 | ./catLps.pl 100 >/dev/null 
1.040

time seq 1 10000 | ./catLps.pl 10000 >/dev/null  
1.042

For fun:
export TIMEFORMAT='%R' ;clear ;time seq 1 $((LINES-2)) | ./catLps.pl $((LINES-2))
